aleph_alpha_client/embedding.py

```diff
@@ -272,6 +269,66 @@ def _asdict(self) -> Mapping[str, Any]:
         return asdict(self)
 
 
+@dataclass(frozen=True)
+class InstructableEmbeddingRequest:
+    """
+    Embeds a text and returns vectors that can be used for classification according to a given instruction.
+
+    Parameters:
+        input
+            The text and/or image(s) to be embedded.
+        instruction
+            An instruction specifying the aspect to attend to when generating the embedding.
+        normalize
+            Return normalized embeddings. This can be used to save on additional compute when applying a cosine similarity metric.
+
+            Note that at the moment this parameter does not yet have any effect. This will change as soon as the
+            corresponding feature is available in the backend
```

Comment:
While I see that this was just C&P, does this still hold true? Is it not yet supported by the backend?

Reply:
I will check. Thanks for pointing that out!